packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -149,6 +187,7 @@ function TableView<T extends object>(props: SpectrumTableProps<T>, ref: DOMRef<H
   let state = useTableState({
     ...props,
     showSelectionCheckboxes,
+    showDragButtons: !!isTableDraggable,
```

Comment:
this is already a boolean
```suggestion
    showDragButtons: isTableDraggable,
```